api-metastore/src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -52,10 +61,11 @@ public SubscriptionController(final SubscriptionService subscriptionService) {
             @RequestParam(value = "limit", required = false, defaultValue = "20") final int limit,
             @RequestParam(value = "offset", required = false, defaultValue = "0") final int offset,
             @RequestParam(value = "token", required = false) final String token,
+            @Nullable @RequestParam(value = "reader", required = false) final AuthorizationAttribute reader,
```

Comment:
AuthorizationAttribute became a view. usually, view model is done via mappings.

Reply:
is this considered mapping https://github.com/zalando/nakadi/pull/1292/files#diff-7348bf9e0c2e92e73a523b0b0cab97c411ab6243f153a9a02783638fb9632de2R51 ?

Reply:
@InitBinde magic is magic, it is not clear how that works. Instead of having view for the user request we are fine to prepare view as a model. In general it can be done, but that AuthorizationAttribute has no relation to `reader` except it has the same fields that's why I prefer to have a separate view and map it to model. I will not argue for that just keep it as it is as you see fit :)